src/chttpd/src/chttpd_misc.erl

```diff
@@ -146,7 +148,7 @@ all_dbs_info_callback({row, Row}, #vacc{resp = Resp0} = Acc) when
     Prepend = couch_mrview_http:prepend_val(Acc),
     case couch_util:get_value(id, Row) of
         <<"_design", _/binary>> ->
-            {ok, Acc};
+            {skip, Acc};
```

Comment:
I think it's a bit confusing that we handle the skip in both the callback (by returning {skip, ...}) and in the caller where we look at skip_design and manage Skip and Limit in

```
                        case Key of
                            <<"_design", _/binary>> -> Skip;
                            _ -> Skip - 1
                        end,
```

We should see if we could take a different approach perhaps.